lib/tweet/_video.dart

```diff
@@ -99,9 +103,34 @@ class _TweetVideoState extends State<TweetVideo> {
     );
   }
 
+  void _loadMuteOptions() {
```

Comment:
I think the content of this method should be inlined in `_loadVideo` - mostly because it otherwise introduces a race, where the video might start playing before the "mute" code runs, meaning audio blares for a moment before stopping.

We should be able to create the controller as normal, but disable `autoPlay`, then set the volume on the controller instance, then start the video playing.

One other thing that will need to be done is to store the `shouldMuteFeed` value inside `_SubscriptionGroupFeedState`, which means passing it down until it gets into this widget - or creating a `Provider` around `_SubscriptionGroupFeedState`, storing the preference in a per-group instance store, which would mean we don't need to pass down through the entire context.